### ISSUE TITLE
Incompatible with Sushi Bar (Library) and (MAYBE) Particular

### ISSUE THREAD

**Issue by Ford3l** [Sinytra/Connector#1253]
### Mod Name

Sushi Bar / Particular

### Mod Homepage

https://modrinth.com/mod/sushi-bar/version/0.1.0               https://modrinth.com/mod/particular

### Minecraft version

1.20.1

### Describe the bug

After launching minecraft, during the download menu, the game crashes

### Steps to reproduce

1. Dowland Particular and Sushi Bar
2. Launch Minecraft
3. Get an error
...

### Logs

https://gist.github.com/Ford3l/59d4b339b72c5f4ae7eb9d4e9062b9ae

**Comment by kakashi1464** [comment]
log says that sushi bar mod is not installed

**Comment by Ford3l** [comment]
Ok? But i installed it

**Comment by Ford3l** [comment]
> log says that sushi bar mod is not installed

Ok? But i installed it

**Comment by kakashi1464** [comment]
check again

**Comment by WitherTM** [comment]
> check again
No, the log says that sushi bar mod is outdated, update it. the log also says the crash is because of an account error, @Ford3l you do have a legitimate account right?

**Comment by Ford3l** [comment]
> > check again
> 
> No, the log says that sushi bar mod is outdated, update it. the log also says the crash is because of an account error, @Ford3l you do have a legitimate account right?

Thanks for the help, I have already solved the problem, another library is needed there.

**Comment by Ford3l** [comment]
another library is needed there, sorry
